src/electionguard_tools/helpers/serialize.py

```diff
@@ -122,6 +123,8 @@ def custom_encoder(obj: Any) -> Any:
         new_dict = asdict(obj)
         obj = _recursive_replace(new_dict, int, _get_int_encoder())
         return obj
+    elif isinstance(obj, BaseElement):
+        return obj.to_hex()
```

Comment:
This is the piece that I was trying to avoid. Having to check for `BaseElement` to do the serialization. This is essentially what we had prior to the switch to `int`.